build/common.props

```diff
@@ -4,7 +4,7 @@
     <Version>3.0.0$(VersionSuffix)</Version>
     <ExtensionsVersion>4.0.1$(VersionSuffix)</ExtensionsVersion> <!-- WebJobs.Extensions -->
     <CosmosDBVersion>3.0.8$(VersionSuffix)</CosmosDBVersion>
-    <HttpVersion>3.0.9$(VersionSuffix)</HttpVersion>
```

Comment:
We should be able to just revert this back to 3.0.8. We haven't released the 3.0.9 version, and I just removed the 3.0.9-xxx version from the staging feed for you.